environments/dev/config.yaml

```diff
@@ -52,7 +52,7 @@ config:
     SHOW_GARDENER_METADATA:
       isEnabled: true
     OBSERVABILITY:
-      isEnabled: true
+      isEnabled: false
```

Comment:
if `OBSERVABILITY` is fully removed please remove all this section from config.

Reply:
That observability is special as it is a real "feature". Removing it fully from the code base was too heavy for me and I like to pass it over to you in a follow-up. So I thought disabling it now will be the safest. I don't know what the default behaviour is when removing the config block only, I can imagine that the default is on.